examples/cephfs/storageclass.yaml

```diff
@@ -47,6 +47,10 @@ parameters:
   # If omitted, defaults to "csi-vol-".
   # volumeNamePrefix: "foo-bar-"
 
+  # (optional) Boolean value. The PVC shall be backed by the snapshot specified
+  # in its data source. (defaults to `false`)
+  # backingSnapshot: "true"
```

Comment:
@gman0 can you add a comment to not set the `pool` Parameter when backingSnapshot is set?